torch/backends/_nnapi/serializer.py

```diff
@@ -1573,6 +1575,30 @@ def add_conv_underscore(self, node):
             NNAPI_FuseCode.FUSED_NONE,
         )
 
+    def add_log_softmax(self, node):
+        assert node.inputsSize() == 3
+        assert node.outputsSize() == 1
+
+        (
+            jit_input,
+            jit_dim,
+            jit_half_to_float
+        ) = node.inputs()
+        input_id, input_oper = self.get_tensor_operand_by_jitval_fixed_size(jit_input)
+        _, dim = self.get_constant_value(jit_dim, "IntType")
+
+        out_shape = input_oper.shape
+
+        inputs = [None] * 3
+        inputs[0] = input_id
+        inputs[1] = self.add_immediate_float_scalar(1)
```

Comment:
Let's add a comment on what this 1 is? "specifying the positive scaling factor for the exponent, beta."